app/code/Magento/AdvancedSearch/Test/Mftf/ActionGroup/AdminSaveNewSearchTermActionGroup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSaveNewSearchTermActionGroup">
```

Comment:
According to the best practices, we usually split big action groups into more granular pieces. Also, you may reuse the existing action group to avoid code duplication. 

For navigating to a page, feel free to use [AdminNavigateMenuActionGroup](https://github.com/magento/magento2/blob/e6b956dff608b7f55830ef78210c7c0a18b79794/app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminNavigateMenuActionGroup.xml#L11)
Or, even better, create an action group that opens the new search term action group in the [same](https://github.com/magento/magento2/blob/70b1ff3b8d4847583a994ac5f2f8f9fad8bb1b31/app/code/Magento/Variable/Test/Mftf/ActionGroup/AdminOpenNewVariablePageActionGroup.xml#L10) manner. Let's call it `AdminOpenNewSerchTermPageActionGroup`

Then we will have two more action groups: 

- `AdminFillSearchTermActionGroup` that accept values for all fields as arguments
- `AdminSaveSearchTermActionGroup` that fill submit the form and wait for the page load

Reply:
@rogyar Great thanks for the code review. I have made changes to the MFTF test based on your recommendations. Please review it.